### DISCUSSION TITLE
如何防止暴力破解？能否添加验证码？

### DISCUSSION THREAD

**Discussion by imasy1024** [lobehub/lobehub#4938] [Self Hosting]
如何防止暴力破解？能否添加验证码？

**Comment by arvinxx** [comment]
暴力破解什么？

**Reply by imasy1024** [reply]
防止暴力破解 ACCESS_CODE

**Comment by Sun-drenched** [comment]
1.用openssl rand -base64 32生成一个高度复杂的ACCESS_CODE
2.配置Clerk、NextAuth等身份验证服务

至于验证码机制，我觉得应该是没有必要的。因为自几个月前lobechat支持服务端数据库和逐步全面接入各种成熟身份验证服务后，ACCESS_CODE从某种意义上已经被淘汰了。

**Comment by BrandonStudio** [comment]
其实可以考虑加个Turnstile之类的？ @arvinxx
